aiven/client/cli.py

```diff
@@ -1198,6 +1198,21 @@ def vpc__peering_connection__request(self):
             print(ex.response.text)
             raise
 
+    def _get_service_project_vpc_id(self):
+        """Utility method for service_create and service_update"""
+        if self.args.project_vpc_id is None:
+            project_vpc_id = None if self.args.no_project_vpc else client.UNDEFINED
+        elif self.args.no_project_vpc:
+            raise argx.UserError("Only one of --project-vpc-id and --no-project-vpc can be specified")
+        else:
+            project_vpc_id = self.args.project_vpc_id
+        return project_vpc_id
+
+    _service_project_vpc_id_help = "Put service into a project VPC. The VPC's cloud must match the service's cloud"
```

Comment:
I wouldn't mind you just using these helps in two places as copypaste, they're unlikely to change and would match the rest of the methods.

Reply:
Changed into passing identical string literals and removing the class attributes